warp-core/src/test/scala/com/workday/warp/persistence/PersistenceUtilsSpec.scala

```diff
@@ -40,7 +40,12 @@ class PersistenceUtilsSpec extends WarpJUnitSpec with CorePersistenceAware {
   @BeforeEach
   def truncateSchema(): Unit = {
     Connection.refresh()
-    CorePersistenceUtils.truncateSchema()
+//    CorePersistenceUtils.truncateSchema()
+    CorePersistenceUtils.dropSchema()
+//    Connection.refresh()
+    // make sure we don't throw an exception when the schema already exists
```

Comment:
sorry, i forgot to tidy this up before telling you to merge in my branch. please delete 43,45, and 46